articles/how-to-apply-post-processing.md

```diff
@@ -0,0 +1,120 @@
+# Post Processing in SadConsole
+
+Sometimes you may want to apply a shader or graphical effect to your game when drawing, which this article will explain how to do. 
+
+To apply a post processing effect to the screen, you can override SadConsole's default drawing functionality, and then apply your own custom draw commands.
+
+To do this, you attach a new `DrawableGameComponent` to SadConsole's game instance components, which is set to draw *after* SadConsole has finished creating the frame for the default screen.
+
+This `DrawableGameComponent` should then access SadConsole's final render surface, and apply whatever custom draw code is needed.
+
+## Method
+
+### Setting up the Class
+To begin with, you'll need to create a new class that inherits from MonoGame's `DrawableGameComponent`, and override the constructor and "Draw" functions so that you can draw to screen.
+
+```
+public class MyCustomPPFX : DrawableGameComponent
+{
+    public MyCustomPPFX() : base(SadConsole.Game.Instance) 
+    {
+        //...
+	}
+
+    //When we need to draw to the screen, it's done here.
+    public override void Draw(GameTime gameTime)
+    {
+
+	}
+}
+```
+
+After this, you'll need to have your constructor load in the shader you want to use. This can be done through two methods:
+
+ 1. The MonoGame Content Pipeline
+ 2. 2MGFX (Manual Inclusion)
+ 
+ The MonoGame Content Pipeline is arguably easier, however requires you to have the pipeline available for changes.
+
+### Loading a Shader
+*Method 1: MonoGame Content Pipeline*
+
+To load your content via. the MonoGame Content Pipeline, you can create a simple `.mgcb` and include the shader as a file. Once this is done, you can set up a content manager and load the shader like so:
+
+```
+effect = Content.Load<Effect> ("myEffect");
+```
+
+*Method 2: 2MGFX and Manual Inclusion*
+
+First, you'll need to compile your `.fx` shader using *2MGFX*, a free tool included in every MonoGame install. The default location for this is in the MonoGame MSBuild folder, located at `Program Files (x86)\MSBuild\MonoGame\Tools`.
+
+> Note: Make sure you're compiling for the correct platform, either DesktopGL or DirectX_11, otherwise your shader will not compile properly, or fail to load.
```

Comment:
You can use the custom docfx note tag to make this render nice:

```suggestion
> [!NOTE]
> Make sure you're compiling for the correct platform, either DesktopGL or DirectX_11, otherwise your shader will not compile properly, or fail to load.
```